<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>daca triage tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="middleLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="CodeEditor" name="code">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="urlFileLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="urllabel">
          <property name="text">
           <string>url</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="urlEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="filelabel">
          <property name="text">
           <string>file</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="fileEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="leftDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="leftDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="leftDockVerticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="loadLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="loadFile">
           <property name="text">
            <string>Load from file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadFromClipboard">
           <property name="text">
            <string>Load from clipboard</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="results"/>
       </item>
       <item>
        <widget class="QCheckBox" name="random100">
         <property name="text">
          <string>Show 100 random results</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="versionLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="versionLabel">
           <property name="text">
            <string>Version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="version"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="rightDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="rightDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="filesNavigatorLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="filterLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLineEdit" name="filterEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>filter</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="inFilesButton">
           <property name="text">
            <string>In files</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="fsTab">
          <attribute name="title">
           <string>Filesystem</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QTreeView" name="directoryTree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="itemsExpandable">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="inFilesTab">
          <attribute name="title">
           <string>In Files Result</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QListWidget" name="inFilesResult">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fileFilterLayout">
             <property name="spacing">
              <number>1</number>
             </property>
             <item>
              <widget class="QCheckBox" name="hFilesFilter">
               <property name="text">
                <string>*.h, *.hpp,...</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="srcFilesFilter">
               <property name="text">
                <string>*.c, *.cpp,...</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>filterEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>fileTreeFilter(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>940</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFilesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>findInFilesClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directoryTree</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>directorytreeDoubleClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFilesResult</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>searchResultsDoubleClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFromClipboard</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFromClipboard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>random100</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>results</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>showResult(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>version</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>filter(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFile()</slot>
  <slot>showResult(QListWidgetItem*)</slot>
  <slot>loadFromClipboard()</slot>
  <slot>filter(QString)</slot>
  <slot>refreshResults()</slot>
  <slot>fileTreeFilter(QString)</slot>
  <slot>findInFilesClicked()</slot>
  <slot>directorytreeDoubleClick()</slot>
  <slot>searchResultsDoubleClick()</slot>
 </slots>
</ui>
